.circleci/config.yml

```diff
@@ -152,6 +152,10 @@ jobs:
       - image: circleci/node:erbium
     steps:
       - checkout
+      - run:
+          name: Install npm 8
+          command: |
+            sudo npm install npm@8.19.2 -g
```

Comment:
It feels like there should be an available docker image that has npm8 pre-installed so we don't need a `run` block here, but I think that our version of Node may be too far out of date (we're pinned to 12.x, it looks like) for images to have updated npm in them, so I'm okay with this approach until/unless we upgrade our Node version.

Reply:
Yeah - I researched this. Even if we were to bump to node 16.x, which is out of scope of this PR, node 16.x only supports npm 7.x

More discussion here: https://github.com/nodejs/node/discussions/37689

Reply:
I hadn't seen that.  Even with that being true, CircleCI (as an example) provides some convenience images that included `npm` 8 along with other versions of `node` (https://circleci.com/developer/images/image/cimg/node), but I don't see any images with `node` 12.x that do that.  And I agree that bumping the `node` version is well beyond the scope of this PR, so this work-around seem reasonable.